### PR TITLE
Rewrite the generator script of unicode ranges.

### PR DESCRIPTION
**Pull request by robertsipka**
The script generates the source file instead of copy the tables and paste these manually.

### CODE REVIEW

File: jerry-core/lit/lit-char-helpers.c [hunk 1]
```diff
@@ -132,9 +132,10 @@ lit_char_is_white_space (ecma_char_t c) /**< code unit */
   {
     return (c == LIT_CHAR_NBSP
             || c == LIT_CHAR_BOM
-            || (c >= unicode_separator_char_interv_sps[0]
-                && c <= unicode_separator_char_interv_sps[0] + unicode_separator_char_interv_lens[0])
-            || search_char_in_char_array (c, unicode_separator_chars, NUM_OF_ELEMENTS (unicode_separator_chars)));
+            || (c >= jerry_unicode_separator_char_interval_sps[0]
+                && c <= jerry_unicode_separator_char_interval_sps[0] + jerry_unicode_separator_char_interval_lengths[0])
+            || search_char_in_char_array (c, jerry_unicode_separator_chars,
```
Comment (zherczeg) [inline jerry-core/lit/lit-char-helpers.c]: Style error, you cannot break arguments this way.

File: jerry-core/lit/lit-unicode-ranges.inc.h [hunk 2]
```diff
@@ -12,199 +12,172 @@
...
+  0x0006, 0x003e, 0x0003, 0x0006, 0x0003, 0x0026, 0x0003, 0x001e, 0x0003, 0x0006,
+  0x0003, 0x0006, 0x0006, 0x0016, 0x001e, 0x0003, 0x0006, 0x0026, 0x0012, 0x0054,
+  0x00ff, 0x00ff, 0x006b, 0x0007, 0x0019, 0x004a, 0x0033, 0x0057, 0x0028, 0x009b,
+  0x0059, 0x0015, 0x0005, 0x0025, 0x0005, 0x0007, 0x001e, 0x0034, 0x0006, 0x0002,
+  0x0006, 0x0003, 0x0005, 0x000c, 0x0002, 0x0006, 0x0009, 0x0004, 0x0003, 0x0002,
+  0x0006, 0x0023, 0x0002, 0x0008, 0x0004, 0x0002, 0x0053, 0x0001, 0x0059, 0x0002,
+  0x0027, 0x005d, 0x0017, 0x00ff, 0x00ff, 0x00ff, 0x00ff, 0x008c, 0x00ff, 0x002d,
+  0x0006, 0x0004, 0x0009, 0x000c, 0x0004, 0x0001, 0x0001, 0x006b, 0x00ff, 0x006a,
+  0x003f, 0x0035, 0x000b, 0x0002, 0x0086, 0x0019, 0x0019, 0x0058, 0x0005, 0x0005,
+  0x0005, 0x0002
```
Comment (zherczeg) [inline jerry-core/lit/lit-unicode-ranges.inc.h]: Why do we use hexadecimal numbers for lengths?

Comment (robertsipka) [inline jerry-core/lit/lit-unicode-ranges.inc.h]: I think it fits better with the other tables.

File: jerry-core/lit/lit-char-helpers.c (hunk 1 above)
Comment (robertsipka) [inline jerry-core/lit/lit-char-helpers.c]: Thanks, I just rewrote the names and haven't checked the style. I've fixed it.

**Pull request merged**